02/app.js

```diff
@@ -1,2 +1,16 @@
 const x = 5;
-let result = 0;
\ No newline at end of file
+let result = 0;
+let i = 0;
+
+for (let i=1; i<=5; i++) {
+    result += i;
+}
+
+// additional task
+
+/* while (i < x) {
+    i++;
+    result += i;
+} */
```

Comment:
Zwróć uwagę na kolejność operacji.
Jeśli `i++` byłoby po obliczeniu `result` to warunek musiałby być inny.
Oczywiście to co zrobiłaś jest ok - chciałem tylko, abyś miała tego świadomość.